h/models/annotation_metadata.py

```diff
@@ -0,0 +1,24 @@
+from h.db import Base, types
+from sqlalchemy.dialects import postgresql as pg
+import sqlalchemy as sa
+from sqlalchemy.ext.mutable import MutableDict
+
+
+class AnnotationMetadata(Base):
+    __tablename__ = "annotation_metadata"
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    annotation_pk = sa.Column(
```

Comment:
I was more sure about `annotation.pk` in the annotation table itself.

Here I wanted to just go for `annotation_id` to future prof the name but we already have tables that relate to annotations by the UUID with a column name `annotation_id`.